controllers/session.go

```diff
@@ -0,0 +1,54 @@
+package controllers
+
+import (
+	"fmt"
+	"net/http"
+
+	"google-scraper/forms"
+
+	"github.com/beego/beego/v2/server/web"
+)
+
+type Session struct {
+	baseController
+}
+
+func (c *Session) NestPrepare() {
+	c.requireGuestUser = true
```

Comment:
@junan Do you planned to use this controller for logout too? If yes then do you have any plan to adjust this to ensure the logout path requires authenticated user in the future PR? I saw Thumb got the same problem also, he moves this to the policy mapping on [his PR](https://github.com/Lahphim/go-crawler-challenge/pull/51).

Reply:
Yes, I will use this controller for logout also. I checked a bit Thumbs changes where he implemented more structural policies with `NestPrepare`. 

It works for for now. But it would be good to have a similar policy approach Thumb doing. I will introduce similar policy structure in my logout task.